lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -301,3 +303,126 @@ def upload_ora2_data(
     TASK_LOG.info(u'%s, Task type: %s, Upload complete.', task_info_string, action_name)
 
     return UPDATE_STATUS_SUCCEEDED
+
+
+def upload_students_grades_data(
```

Comment:
As for me, will be great, if doc stings will contain schema information

Reply:
Oh, and seems you forget to update it completely

Reply:
Util function that generates students grades contain schema information in the docstring